node/network/statement-distribution/src/vstaging/candidates.rs

```diff
@@ -138,11 +139,12 @@ impl Candidates {
 		Ok(())
 	}
 
-	/// Note that a candidate has been confirmed. If the candidate has just been
-	/// confirmed, then
+	/// Note that a candidate has been confirmed.
+	///
+	/// If the candidate has just been confirmed, then remove any outdated claims, and generate a
+	/// reckoning of which peers advertised correctly and incorrectly.
```

Comment:
Good point, updated the comment. I didn't remove it because I find it helpful -- while it's there in the code, a high-level, condensed description of the intent is still good to have IMO.